### DISCUSSION TITLE
npm i -D cy-verify-downloads returns error  error cb() never called

### DISCUSSION THREAD

**Discussion by aichingerstefanie** [community/community#46154] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Dear community users! 
I use cypress for writing software tests.  I want to test file download per ling and found cy-verify-downloads but neither download nor installation does work. 
Can anybody help me with this issue: ( proxyname and portnumber already verified in my company)
command executed: npm i -D cy-verify-downloads --proxy <here is my proxyname and portnumber>
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'i',
1 verbose cli   '-D',
1 verbose cli   'cy-verify-downloads',
1 verbose cli   '--proxy',
1 verbose cli   '<here is my proxyname and portnumber>.'
1 verbose cli ]
2 info using npm@6.14.8
3 info using node@v14.15.1
4 verbose npm-session a9396455d6cccd3b
5 silly install loadCurrentTree
6 silly install readLocalPackageData
7 http fetch GET 407 https://registry.npmjs.org/cy-verify-downloads 56ms
8 timing npm Completed in 1197ms
9 error cb() never called!
10 error This is an error with npm itself. Please report this error at:
11 error <https://npm.community>

**Comment by ghost** [comment]
The error message "cb() never called" typically occurs when a callback function has been passed to a function but it has not been executed. It seems like in this specific instance, the error is happening during an installation of the `cy-verify-downloads` package using npm.

This error could be caused by a few things such as a bug in the package, a conflict with another installed package, or an issue with the version of npm being used. To resolve the issue, you could try the following:

 1. Updating npm (worked for me)
 2. Clearing the cache: Run `npm cache clean --force` to clear the cache and try installing the package again. 
 3. Installing the package with the `--no-optional` flag: `npm i -D cy-verify-downloads --no-optional`
4. Reinstalling Node.js: Uninstall and then reinstall Node.js. (wouldn't recommend)]

**Reply by aichingerstefanie** [reply]
Thank you very much for your quick reply! After all, it seems to keep beeing an error in network proxy authentication and cb() never called might be sideeffect or subsequent error. We will try all in other network. Thank you very much!
